### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by krate27**
Screenshot (559)
Screenshot (560)

### CODE REVIEW

**Comment by prath-dev** [comment]
Clean And Elegant UI

File: CSS-files-e-com/login.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+.eye-btn{
+    position: absolute;
+    top: 4.1rem;
+    right: .5rem;
+    border: none;
+}
+.btn-login{
+    padding: .4rem;
+    border-radius: .4rem;
+    /* outline: none; */
```
Comment (RakshithBhat03) [inline CSS-files-e-com/login.css:37]: You can remove this comment if not needed

File: CSS-files-e-com/login.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+.login-test:hover{
+    background-color: var(--danger-message-color);
+    color: var(--white-color);
+}
+.login{
+    background-color: var(--danger-message-color);
+    color: var(--white-color);
+}
+.login:hover{
+    background-color: #ff3300;
```
Comment (RakshithBhat03) [inline CSS-files-e-com/login.css:54]: You can use a variable here

File: CSS-files-e-com/signup.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.input-id-mandatory:after{
+    content: "*";
+    color: var(--primary-color);
+}
+.input-flex{
+    display: flex;
+    width: 20rem;
+    justify-content: space-between;
+    /* align-items: center; */
```
Comment (RakshithBhat03) [inline CSS-files-e-com/signup.css:9]: Same here

File: Components-E-Com/Home-Page/index.html [hunk 4]
```diff
@@ -51,41 +51,41 @@ <h2 class="text-align-center top-margin-7rem"> Featured Categories </h2>
...
-                <div class="card-container">
-                    <main class="main-body wrap-up">
-                        <h3 class="changed-position card-title bolder">Clothing</h3>
-                        <picture class="background-img">
-                            <img src="/Assets/category-1.jpg" alt="image" class="basic-image">
+                <div class="card-contain-e-land">
+                    <main class="wrap-up main-body-e-land">
+                        <h3 class="bolder change-pos-e-land card-title-e-land">Clothing</h3>
+                        <picture class="background-img-e-com">
+                            <img src="/Assets/category-1.jpg" alt="image" class="basic-image-e-land">
```
Comment (RakshithBhat03) [inline Components-E-Com/Home-Page/index.html:58]: `alt` should be more descriptive. Change this throughout the code.

**Review by RakshithBhat03 — COMMENTED** [review]
Suggested small changes. Everything else looks fine

File: CSS-files-e-com/login.css (hunk 1 above)
Comment (krate27) [reply CSS-files-e-com/login.css:37]: have kept it for future reference

File: CSS-files-e-com/filter.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    display: flex;
+}
+
+.all-card-collection {
+    margin: 7rem 3rem 0rem;
+}
+
+.basic-image {
+
+    width: 100%;
```
Comment (abhijeetekad) [inline CSS-files-e-com/filter.css:39]: Remove these blank lines.

**Pull request merged**